lib/deaf_grandma.rb

```diff
@@ -14,13 +14,27 @@ def run!
     loop do
       user_input = get_user_input
       p speak(user_input)
+      if @bye_counter > 0
+        exit(0)
+      end
     end
   end
 
 
   def speak(input)
 
-    #Implement your code here <<<<<<<<<
+    response = "SPEAK UP SONNY!"
+
+    if input.upcase!.nil?
```

Comment:
Im not sure I understand. Here's the source. 
http://ruby-doc.org/core-2.1.0/String.html#method-i-upcase